conans/test/integration/toolchains/test_cmake.py

```diff
@@ -164,6 +164,20 @@ def _run_app(self, build_type, bin_folder=False, msg="App", dyld_path=None):
         self.assertIn("MYDEFINE_CONFIG: MYDEF_%s" % build_type.upper(), self.client.out)
 
 
+class TestNew(unittest.TestCase):
+    def test_new(self):
+        client = TestClient()
+        client.run("new pkg/0.1 --template=v2_cmake")
+        conanfile = client.load("conanfile.py")
+        self.assertIn("CMakeToolchain", conanfile)
+        conanfile = client.load("test_package/conanfile.py")
+        self.assertIn("CMakeToolchain", conanfile)
+        cmake = client.load("test_package/CMakeLists.txt")
+        self.assertIn("find_package", cmake)
+        client.run("create .")
+        self.assertIn("pkg/0.1: Hello World Release!", client.out)
```

Comment:
Maybe we can move this test together with other tests of the `conan new` command (`test/functional/command/new_test.py`)

Reply:
If it doesn't build, yes, sure, I just wanted to check that the package is actually building, etc, including the ``test_package()``, this hasn't been test in the toolchain so far. But as it will be used by other tests soon, yep, lets move it (and remove the actual build)

Reply:
Done